priv/templates/extended_bin

```diff
@@ -163,14 +163,22 @@ PIPE_DIR="${PIPE_DIR:-/tmp/erl_pipes/$NAME/}"
 
 # Extract the target cookie
 COOKIE_ARG="$(grep '^-setcookie' "$VMARGS_PATH" || true)"
+DEFAULT_COOKIE_FILE="$HOME/.erlang.cookie"
 if [ -z "$COOKIE_ARG" ]; then
-    echo "vm.args needs to have a -setcookie parameter."
-    exit 1
+    if [ ! -f "$DEFAULT_COOKIE_FILE" ]; then
+        echo "vm.args needs to have a -setcookie, or $DEFAULT_COOKIE_FILE is required."
+        exit 1
+    elif [ -O "$DEFAULT_COOKIE_FILE" ] && [ $(stat --format=%a $DEFAULT_COOKIE_FILE) -eq 400 ]; then
```

Comment:
I wonder if we really need to check the file permissions. For one thing, I could see the file being created by hand and having different permissions, and as long as the file is readable, I don't think it's relevant what it's specific permissions are. If we keep it, I think we also need to make sure `--format=%a` is a valid flag on platforms like Solaris, as I know a few people build releases for that (or some variant of it).